tests/dast/rapid-lca/test_oobt.py

```diff
@@ -27,26 +14,14 @@ def test_oobt_basic():
     RAPIDAST_CFG_FILE = "v5-none-oobt-template.yaml"
 
     port = random.randint(10000, 30000)
-    ipaddr = get_vpn_ip_address()
 
-    # create a rapidast config
-    sed_cmd = f"sed 's/-p <port> -i <ipaddr>/-p {port} -i {ipaddr}/' {TEST_DATA_DIR}/{RAPIDAST_CFG_FILE} > rapidast_runtime_cfg.yaml"
-    os.system(sed_cmd)
-    
     # prep for testing
     os.system(f"chmod 666 {TEST_DATA_DIR}/kubeconfig")
-    if not os.path.exists("results"):
-        os.makedirs("results")
-        os.system("podman unshare chown 1000 results")
 
-    # Run the command and capture stdout
-    command = f"podman run -it --rm -v ./{TEST_DATA_DIR}/kubeconfig:/home/rapidast/.kube/config:Z -v ./results:/opt/rapidast/results:Z -v $PWD:/test:Z -p {port}:{port} {RAPIDAST_IMAGE} rapidast.py --config /test/rapidast_runtime_cfg.yaml"
-    print(command)
-
-    process = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
```

Comment:
It could be better to add here try & except block, so we catch the error if any occurs.
(there is no error handling on the stderr)